test/e2e/performanceprofile/functests/2_performance_update/memorymanager.go

```diff
@@ -105,13 +106,11 @@ var _ = Describe("[rfe_id: 43186][memorymanager] Memorymanager feature", func()
 
 			if !*profile.Spec.RealTimeKernel.Enabled {
 				profile.Spec.RealTimeKernel = &performancev2.RealTimeKernel{
-					Enabled: pointer.BoolPtr(true),
+					Enabled: pointer.Bool(true),
 				}
 			}
-			if profile.Spec.WorkloadHints == nil {
-				profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
-					RealTime: pointer.Bool(true),
-				}
+			profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
```

Comment:
so we are always forcing the hints to this value. Is that what we really need to do? Is it a problem if we overwrite other hints?

Reply:
Right because if we set RealTime to true, then having a profile with Realtime performance hint being disabled would cause issues, So we wanted to make it default to have Realtime workload hint to be true, other workload hints are not an issue.

Reply:
Provided we restore the profile correctly if needed, and we don't pollute any global state (e.g. is the profile instance being changed unique to the testcase or shared among testcases?), fine for me.